build.gradle.kts

```diff
@@ -1,16 +1,27 @@
 buildscript {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
     dependencies {
-        classpath("org.jetbrains.kotlinx:kover:0.6.1")
+        classpath(Dependency.KOTLIN_GRADLE_PLUGIN)
+        classpath(Dependency.GRADLE)
+        classpath(Dependency.KOTLIN_SERIALIZATION)
+        classpath(Dependency.BUILD_KONFIG)
+		classpath(Dependency.KOVER)
     }
 }
 
 plugins {
-    //trick: for the same plugin versions in all sub-modules
-    id("com.android.application").version("7.4.2").apply(false)
-    id("com.android.library").version("7.4.2").apply(false)
-    id("org.jetbrains.kotlinx.kover") version "0.6.1"
-    kotlin("android").version("1.8.0").apply(false)
-    kotlin("multiplatform").version("1.8.0").apply(false)
+    id(Plugin.KOVER_FULL_PATH) version (Version.KOVER)
+}
+
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
 }
```

Comment:
How about we add the JSON API dependency using the Gradle script instead of manually adding the module? Would be easier to manage the versions later on IMO. Or is there a quick automated way to add it to the project? 🤔 
```suggestion
allprojects {
    val buildProperties = rootDir.loadGradleProperties("buildKonfig.properties")
    repositories {
        google()
        mavenCentral()
        maven {
            name = "Github Packages"
            url = uri("https://maven.pkg.github.com/nimblehq/jsonapi-kotlin")
            credentials {
                username = buildProperties.getProperty("GITHUB_USER")
                password = buildProperties.getProperty("GITHUB_TOKEN")
            }
        }
    }
}
```

Reply:
Let me try this, I faced issues with version compatibility with JSON API, so I decided to fork to my repo and solve the issues. We also set our app minSdk = 21 but in the JSON API, it is 23.

<img width="665" alt="Screenshot 2023-04-21 at 10 11 38" src="https://user-images.githubusercontent.com/22606906/233531661-e211ed3c-58f6-4a0b-831e-5864ab12cc1f.png">